packages/orchestration/src/exos/local-chain-facade.js

```diff
@@ -38,17 +41,25 @@ const prepareLocalChainFacadeKit = (
   zone,
   {
     makeLocalOrchestrationAccountKit,
+    agoricNames,
     localchain,
     // TODO vstorage design https://github.com/Agoric/agoric-sdk/issues/9066
     // consider making an `accounts` childNode
     storageNode,
-    vowTools: { allVows, watch },
+    vowTools: { allVows, watch, asVow },
   },
 ) =>
   zone.exoClassKit(
     'LocalChainFacade',
     {
-      public: ChainFacadeI,
+      public: M.interface('LocalChainFacade', {
+        getChainInfo: M.call().returns(VowShape),
+        makeAccount: M.call().returns(VowShape),
+        getVBankAssets: M.call().optional(M.boolean()).returns(VowShape),
```

Comment:
to add this to the types, I the most direct way to add an `extends 'agoric'` case to `ActualChainInfo` https://github.com/Agoric/agoric-sdk/blob/a724e4ace9e70a58a3546a9f80f41f9489da496e/packages/orchestration/src/exos/chain-hub.js#L20-L28